resources/public/api/conf/1.0/application.yaml

```diff
@@ -1,47 +1,536 @@
-openapi: "3.0.3"
-
+openapi: 3.0.3
 info:
-  version: 1.0.0
-  title: Hello World
-
+  title: Excise Movement Control System API (Discovery/Proposal)
+  description: |-
+    This is a very rough draft of an idea for a potential replacement for the current [EMCS SOAP API](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/xml/Excise%20Movement%20Control%20System)
+  contact: {}
+  version: "0.0.0"
 servers:
-  - url: https://test-api.service.hmrc.gov.uk/hello-world
+  - url: https://test-api.service.hmrc.gov.uk
     description: Sandbox
-  - url: https://api.service.hmrc.gov.uk/hello-world
+  - url: https://api.service.hmrc.gov.uk
     description: Production
-
-components:
-  parameters:
-    acceptHeader:
-      name: Accept
-      in: header
-      schema:
-        type: string
-        enum: [
-          "application/vnd.hmrc.1.0+json",
-          "application/vnd.hmrc.1.0+xml"
-        ]
-      required: true
-
+tags:
+  - name: Movements
+    description: Create, amend or cancel an Excise Movement
+  - name: Traders
+    description: Information relating to Excise Traders
 paths:
-  /hello-world:
+  /customs/excise/movements:
+    post:
+      tags:
+        - Movements
+      summary: Submit a new Draft Excise Movement
+      description: Submit a new Draft Excise Movement. </br></br>**Existing SOAP API equivalent:** SubmitDraftMovement </br></br>**Note:** You can only submit a Draft Excise Movement if the Consignor Id matches an Excise Number in the bearer token that you use for API authentication. Otherwise a 403 Forbidden error is thrown.
+      operationId: submitMovement
+      requestBody:
+        required: true
+        content:
+          application/xml:
+            schema:
+              $ref: '#/components/schemas/IE815'
+            examples:
+              submitDraftMovement:
+                summary: Submit draft movement (IE815)
+                value: '<IE815><!-- Example IE815 message --></IE815>'
+      responses:
+        '202':
+          description: Accepted
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExciseMovement'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/ExciseMovement'
+        '403':
+          description: No ERN in authentication token
```

Comment:
Use of ERN acronym which hasn't been defined above?

Reply:
I have now described the acronym and also added some acronyms to the README.md